src/FakeItEasy/Configuration/RuleBuilder.cs

```diff
@@ -63,7 +63,6 @@ public IVoidConfiguration WhenArgumentsMatch(Func<ArgumentCollection, bool> argu
 
         public virtual IAfterCallSpecifiedConfiguration DoesNothing()
         {
-            this.RuleBeingBuilt.Applicator = x => { };

```

Comment:
There's another reason why this change doesn't really work : it no longer causes the call to do nothing if the call was doing something... With your current fix, if you do something like this:

```
var fake = A.Fake<SomeClass>();
A.CallTo(fake).CallsBaseMethod();
A.CallTo(() => fake.SomeMethod()).DoesNothing();
```
`DoesNothing` will _not_ prevent the call to `base.SomeMethod()`.

I think `DoesNothing` should reset the applicator to its initial value (the one set in the `BuildableCallRule` constructor).